manager/orchestrator/update/updater.go

```diff
@@ -616,25 +623,32 @@ func (u *Updater) rollbackUpdate(ctx context.Context, serviceID, message string)
 	}
 }
 
-func (u *Updater) completeUpdate(ctx context.Context, serviceID string) {
+func (u *Updater) completeUpdate(ctx context.Context, serviceID string, force bool) {
 	log.G(ctx).Debugf("update of service %s complete", serviceID)
 
 	err := u.store.Update(func(tx store.Tx) error {
 		service := store.GetService(tx, serviceID)
-		if service == nil {
+		switch {
+		case service == nil:
 			return nil
-		}
-		if service.UpdateStatus == nil {
+		case service.UpdateStatus == nil && force:
+			// Force marking the status as updated; to account for annotation-only updates.
+			service.UpdateStatus = &api.UpdateStatus{
+				StartedAt: ptypes.MustTimestampProto(time.Now()),
```

Comment:
Not sure if this was really needed (setting `StartedAt`), but thought it would be "ok" for consistency. Happy to remove though